docs/developer-onboarding.md

```diff
@@ -2,22 +2,12 @@
 
 ## Purpose
 
-This document describes the on-boardng steps that need to be undertaken for new Developers when they join the team.
+This document describes the on-boardng steps for new Developers when they join the team.
 
-## Azure/DevOps steps
+## PaaS/DevOps steps
 
-These steps can only be completed by members of the DevOps community. At the present time the contacts for this are Colin Saliceti, Vignesh Natraj or Karthik Kumar.
+### Get a PaaS account
+You can get an account by requesting one in the `#digital-tools-support` Slack channel. Request an account with a SpaceDeveloper role for your @digital.education.gov.uk email address with access to the `dfe` organisation and `bat-prod`, `bat-staging` and `bat-qa` spaces.
 
```

Comment:
Should this page have a quick tutorial on PaaS? Or should it have links to a good resource for learning PaaS?

Reply:
A link to https://docs.cloud.service.gov.uk/ would be helpful, but I think a cheatsheet with some common commands and a mention of `make <env_name> shell` would be a nice addition as well.